plugins/pa11y/readme.md

```diff
@@ -0,0 +1,36 @@
+# Description 
+ Pa11y is your automated accessibility testing pal. It runs accessibility tests on your pages via the command line or Node.js, so you can automate your testing process. Pa11y runs automatically on the `test:review` and `test:staging` hooks, using the information about the app from the state.
+
+# Pa11y documentation
+ `https://github.com/pa11y/pa11y`
+
+# Pa11y configurations added on the project
+    Pa11ySchema = {
+        - host: Domain against pa11y will run,
+        - wait: The time in milliseconds to wait before running HTML CodeSniffer on the page,
+        - exceptions: An array of result codes and types that you'd like to ignore. You can find the codes for each rule in the console output and the types are error, warning, and notice. Note: warning and notice messages are ignored by default,
+        - hide: A CSS selector to hide elements from testing, selectors can be comma separated. Elements matching this selector will be hidden from testing by styling them with visibility: hidden,
```

Comment:
issue: do you mean `hideElements` here? (we might need to change hide to hideElements in the pa11ySchema too)

Reply:
question: is there a reason why you've reverted `hideElements` back to `hide` in `schema/pa11y.ts`?